war/src/main/js/widgets/variables.less

```diff
@@ -1,2 +1,27 @@
+@brightest::#fff;
+@birghtest:#fff;
+@bright:#f9f9f9;
```

Comment:
I do, but since I am making up my color scheme as I go along, I am not sure yet what gradients levels I need or want.